### DISCUSSION TITLE
30/4

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#85] [Daily Sprint]
Buen día
Qué hice ayer? 
Que estoy haciendo? 
Impedimento?

**Comment by Pamela198713** [comment]
Ayer tuve clases. 
Hoy tengo una capacitación en el colegio por lo que voy ausentarme varias hs.
Impedimento: organización

**Comment by LeonardoEC** [comment]
Ayer: Trabaje todo el día
Hoy: Planeo organizarme con los estudios
Impedimento: Necesito invertir el tiempo en cosas rentables económicamente

**Comment by rpgrca** [comment]
Hoy no hice mucho del proyecto, tuve reunión presencial en el trabajo. A la noche estuve en la clase de Emprendedurismo. Mañana empiezo con la parte web.
